fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -2583,6 +2583,10 @@ public static void getDdlStmt(String dbName, Table table, List<String> createTab
                         .append("\" = \"");
                 sb.append(storageProperties.get(PropertyAnalyzer.PROPERTIES_STORAGE_CACHE_TTL)).append("\"");
 
+                sb.append(StatsConstants.TABLE_PROPERTY_SEPARATOR).append(PropertyAnalyzer.PROPERTIES_STORAGE_VOLUME)
+                        .append("\" = \"");
+                sb.append(storageProperties.get(PropertyAnalyzer.PROPERTIES_STORAGE_VOLUME)).append("\"");
```

Comment:
if the storage volume name is changed, is it necessary to return the latest name?

Reply:
We can support it later by getting storage volume by id to get the latest name.